pkg/experiment/metadata.go

```diff
@@ -35,30 +35,41 @@ const (
 // MetadataConfig encodes the settings for connecting to the database.
 type MetadataConfig struct {
 	CassandraAddress           string
-	CassandraPort              int
-	CassandraUsername          string
-	CassandraPassword          string
 	CassandraConnectionTimeout time.Duration
-	CassandraSslEnabled        bool
-	CassandraSslHostValidation bool
+	CassandraCreateKeyspace    bool
+	CassandraIgnorePeerAddr    bool
+	CassandraInitialHostLookup bool
+	CassandraKeyspaceName      string
+	CassandraPassword          string
+	CassandraPort              int
 	CassandraSslCAPath         string
 	CassandraSslCertPath       string
+	CassandraSslEnabled        bool
+	CassandraSslHostValidation bool
 	CassandraSslKeyPath        string
+	CassandraTimeout           time.Duration
+	CassandraUsername          string
 }
 
 // DefaultMetadataConfig returns a setup which use a Cassandra cluster running on localhost
 // without any authentication or encryption.
 func DefaultMetadataConfig() MetadataConfig {
```

Comment:
Are those defaults differs from `MetadataConfigFromFlags`? If not, maybe we could drop one of them?